doc/source/guide/installation/index.rst

```diff
@@ -20,7 +20,9 @@ Installing Scientific Python and SunPy
 If you do not currently have a working scientific Python distribution this
 guide will set you up with the Anaconda Python distribution. Alternative options
 exist for different operating systems and can be found later in the
-:ref:`advanced-install` section.
+:ref:`advanced-install` section.Sunpy currently supports Python 2.7 with Anaconda.
+The documentation is designed for Python 2.7. Support for Python 3.x would be
```

Comment:
I'd suggest changing the the Python 3.x sentence to "Support for Python 3.x is coming soon".